src/main/frontend/handler/export.cljs

```diff
@@ -26,12 +26,6 @@
    (outliner-tree/blocks->vec-tree
     (db/get-page-blocks-no-cache repo page) page) {:init-level 1}))
 
-(defn- get-page-content-debug
```

Comment:
@tiensonqin I removed this but if it's useful to have around I could comment it out